### DISCUSSION TITLE
Multiplayer saving game progress

### DISCUSSION THREAD

**Discussion by realbout** [diasurgical/DevilutionX#1581] [Q&A] [answered]
Hello there,

thanks for this wonderful project. Diablo is one of the best games ever and you guys bring it back to life!
Its been a while since i last played Diablo in multiplayer. Is there a way to save the ingame progress or did we have to restart from level 1? I don't mean the character level and items, these are saved fine so far. I mean the exploration of the world.

Thanks in advance!

**Comment by AJenbo** [comment, marked as answer]
Yeah the world resets between each game in multiplayer but all shortcuts are always open

**Reply by realbout** [reply]
Thanks a lot! This answers any further questions about that.
